exporter/rabbitmqexporter/factory.go

```diff
@@ -109,7 +118,11 @@ func createLogsExporter(
 	config := (cfg.(*Config))
 
 	routingKey := getRoutingKeyOrDefault(config, logsRoutingKey)
-	r := newRabbitmqExporter(config, set.TelemetrySettings, newPublisherFactory(set), newTLSFactory(config), routingKey, logsConnectionName)
+	connectionName := defaultLogsConnectionName
+	if config.Connection.Name != "" {
+		connectionName = config.Connection.Name
+	}
+	r := newRabbitmqExporter(config, set.TelemetrySettings, newPublisherFactory(set), newTLSFactory(config), routingKey, connectionName)
```

Comment:
looks like a straightforward change, any easy test to support it and keep coverage?